src/store/houses/actions.ts

```diff
@@ -5,11 +5,16 @@ import { HTTP } from '@/core/api/http-common';
 
 export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Actions = {
   [HousesActionsEnum.SET_HOUSES]: ({ commit }) => {
-    HTTP.get(`/cooperations/1/houses`).then((r) => {
+    HTTP.get(`/cooperations/553/houses`).then((r) => {
```

Comment:
This hardcoded value is temporary, right?

Reply:
It was hard code in another component with value 1. I changed corporations because we had problems with team members who works with deleting and adding houses functionality.

Reply:
In next commit I fixed my action 'GET_HOUSE_BY_ID' by using store getter 'getSelectedCooperationId '.
Is it better way?